### PR TITLE
Update deploy.py

### PR DESCRIPTION
**Pull request by mikhail**

### CODE REVIEW

**Comment by mikhail** [comment]
Review? @swaggy

File: kingpin/bin/deploy.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def main():
     if args.build_only:
         try:
             get_main_actor(dry=False)
-        except actor_exceptions.ActorException as e:
+        except Exception as e:
```
Comment (diranged) [inline kingpin/bin/deploy.py]: Why this change?

Comment (mikhail) [inline kingpin/bin/deploy.py]: Because if something raises an exception that wasn't caught and converted to a kingpin exception then this code doesn't catch it and exits with status 0.

For example - yaml parser raised ScannerException which was not caught by anything

**Comment by swaggy** [comment]
👍 Nice 👍

File: kingpin/bin/deploy.py (hunk 1 above)
Comment (diranged) [inline kingpin/bin/deploy.py]: That was an intentional decision by you and I back in the day.. the idea was that any uncaught exception _DID_ crash the app on purpose because its not expected.

Comment (diranged) [inline kingpin/bin/deploy.py]: I should add, this can be rethought ... but it was intentional at the time.

Comment (mikhail) [inline kingpin/bin/deploy.py]: It still crashes the app ... I explicitly log it and exit with status 1. The problem is that if python exists on its own - then it's status 0, and that's bad.

**Comment by diranged** [comment]
👍 lets just merge this.. seems fine i think.
